### DISCUSSION TITLE
請問有6.2日文原文的CSV嗎?

### DISCUSSION THREAD

**Discussion by tenchi6707** [GpointChen/FFXIVChnTextPatch-GP#219] [General]
大大好，請問有6.2日文原文的CSV嗎?
謝謝!

**Comment by GpointChen** [comment]
https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.67/rawexd-ja-620.zip

**Comment by tenchi6707** [comment]
謝謝大佬!
